deployments/goerli/usdt/migrations/1691022234_configurate_and_ens.ts

```diff
@@ -0,0 +1,141 @@
+
+    if (officialMarketsJSON[arbitrumChainId]) {
+      officialMarketsJSON[arbitrumChainId].push(newMarketObject);
+    } else {
+      officialMarketsJSON[arbitrumChainId] = [newMarketObject];
+    }
+
+    const actions = [
+      // 1. Set the factory in the Configurator
+      {
+        contract: configurator,
+        signature: 'setFactory(address,address)',
+        args: [comet.address, cometFactory.address],
+      },
+
+      // 2. Set the configuration in the Configurator
+      {
+        contract: configurator,
+        signature: 'setConfiguration(address,(address,address,address,address,address,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint64,uint104,uint104,uint104,(address,address,uint8,uint64,uint64,uint64,uint128)[]))',
+        args: [comet.address, configuration],
+      },
+
+      // 3. Deploy and upgrade to a new version of Comet
+      {
+        contract: cometAdmin,
+        signature: "deployAndUpgradeTo(address,address)",
+        args: [configurator.address, comet.address],
+      },
+
+      // 4. Set the rewards configuration to COMP
+      {
+        contract: rewards,
+        signature: "setRewardConfig(address,address)",
+        args: [comet.address, COMPAddress],
+      },
+
+      // 5. Set the official markets text record on the subdomain
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, officialMarketsJSON]
+        )
+      }, 
+
+      // 6. Send USDT from Timelock to Comet
+      // XXX assert that funds have been transferred by diffing the balances before and after
+      {
+        contract: USDT,
+        signature: "transfer(address,uint256)",
+        args: [comet.address, exp(5, 6)],
+      },
+    ];
+
+    const description = "# Initialize cUSDTv3 on Goerli"
+    const txn = await deploymentManager.retry(
+      async () => trace((await governor.propose(...await proposal(actions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  }, 
+
+  async verify(deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager, preMigrationBlockNumber: number) {
+    const ethers = deploymentManager.hre.ethers;
+    await deploymentManager.spider(); // Pull in Arbitrum COMP now that reward config has been set
+    const {
+      comet,
+      rewards,
+    } = await deploymentManager.getContracts();
+
+    const config = await rewards.rewardConfig(comet.address);
+
+    // 1. Verify state changes
+    const stateChanges = await diffState(comet, getCometConfig, preMigrationBlockNumber);
+    // expect(stateChanges).to.deep.equal({
```

Comment:
Will uncomment once the first initial comet is deployed. 
Before contract is actually deployed, this verify will fail.

Reply:
Uncommented as the contract has been depoyed.